k8s/charts/openebs/templates/deployment-local-provisioner.yaml

```diff
@@ -54,7 +57,7 @@ spec:
         - name: OPENEBS_IO_BASE_PATH
           value: "{{ .Values.localprovisioner.basePath }}"
         - name: OPENEBS_IO_HELPER_IMAGE
-          value: "{{ .Values.localprovisioner.helperImage }}"
+          value: "{{ .Values.localprovisioner.helperImage }}:{{ .Values.localprovisioner.helperImageTag }}"
```

Comment:
Here `helperImage` also contains the tag info as well, but later we changed while raising the PR to upstream stable charts, please pull the values files as well from stable/charts.

Reply:
Done